.env.dist

```diff
@@ -1,9 +1,11 @@
-# This file is a "template" of which env vars need to be defined for your application
-# Copy this file to .env file for development, create environment variables when deploying to production
-# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+# Symfony application's path (absolute or relative)
+SYMFONY_APP_PATH=./app
 
-###> symfony/framework-bundle ###
-APP_ENV=dev
```

Comment:
Why these settings were removed?

Reply:
They didn't removed. Symfony installation moved to app folder and there's a new .env.dist for the entire project.